core/Tracker/VisitExcluded.php

```diff
@@ -89,8 +89,11 @@ public function isExcluded()
          * @param bool &$excluded Whether the request should be excluded or not. Initialized
          *                        to `false`. Event subscribers should set it to `true` in
          *                        order to exclude the request.
+         * @param Request $request Additional Informations for plugins to decide, if the visit is excluded
```

Comment:
Can you please write more description ie `The request object which contains all of the request's information`